exercises/concept/paint-by-number/.docs/instructions.md

```diff
@@ -0,0 +1,159 @@
+# Instructions
+
+[Paint by number][paint-by-number] (also known as _color by number_)
+are black-and-white pictures meant for coloring.
+The different areas of the picture are annotated with different numbers,
+the numbers correspond to specific colors in a predefined color palette.
+The goal is to fill the areas with the right colors,
+revealing a beautiful colorful picture at the end.
+It's a relaxing activity for both kids and adults.
+
+You have been tasked with writing a paint-by-number app in Elixir.
+You want your app to be able to import and export pictures in a custom data format.
+You have decided to use [binary files][binary-file] to store your picture data.
+
+~~~~exercism/note
+This exercise assumes that you're familiar with [binary numbers](https://en.wikipedia.org/wiki/Binary_number)
+and understand the principles behind changing binary numbers to decimal numbers
+and decimal numbers to binary numbers.
+~~~~
+
+Let's imagine you have a picture of a heart, like the one represented by ASCII art below.
+The picture has a white background. The heart has a black border and a red fill color.
+
+```
+    * * *       * * *
+  * . . . *   * . . . *
+* . . . . . * . . . . . *
+* . . . . . . . . . . . *
+* . . . . . . . . . . . *
+* . . . . . . . . . . . *
+  * . . . . . . . . . *
+    * . . . . . . . *
+      * . . . . . *
+        * . . . *
+          * . *
+            *
+```
```

Comment:
I'm repeating myself, high-school sweetheart already has an ASCII heart 😉 but a heart was the most recognizable shape I could do in ASCII that could be made out of "pixels" and not infinitely many arbitrary text characters like ASCII art usually is.

I saw that the sudoku killer exercise comes with an image. I think I will make this into an image, with some different shape... like a smiley? Or this bee https://ih1.redbubble.net/image.93120918.6282/flat,800x800,075,f.jpg ?

Reply:
Am good to follow your intuition here. I think the heart is good even though we have it somewhere else as it is simple and not likely to be mistaken or misinterpreted.